README.md

```diff
@@ -91,6 +87,10 @@ Have a look at our [contributing guidelines][contributing.guidelines] to find ou
 
 * [Adyen API Explorer][docs.apiExplorer]
 
+* [UI Customization guide][docs.github.uiCustomization]
+
+* [Additional documentation for payment methods][docs.github.paymentMethods]
```

Comment:
Clicking on the link currently shows this page:

![Screenshot 2024-09-19 at 10 52 27](https://github.com/user-attachments/assets/69308164-bde4-4b34-b783-ceaa3639a10d)

It's obvious what each page means from the name but maybe it can look nicer if we add a `README.md` inside `/docs/payment-methods` and we list the contents nicely. WDYT?

Reply:
Will you do this in this PR or another?

Reply:
I will do it in this PR